File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/SessionListMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

import de.hsrm.mi.swtpro03.server.game.dtos.SessionDTO;

import java.util.List;
import java.util.Map;

public class SessionListMessage {

	public static final String TYPE = "SessionListMessage";

	private List<SessionDTO> listOfSessionDTOS;

	public SessionListMessage() {
	}

	public SessionListMessage(List<SessionDTO> listOfSessionDTOS) {
		this.listOfSessionDTOS = listOfSessionDTOS;
	}

	public List<SessionDTO> getListOfSessionDTOS() {
		return listOfSessionDTOS;
	}

	public void setListOfSessionDTOS(List<SessionDTO> listOfSessionDTOS) {
		this.listOfSessionDTOS = listOfSessionDTOS;

	}
}

File: client/Client/Network/Messages/Session/SessionListMessage.cs
using LoadRunnerClient.DTOs;
using System.Collections.Generic;

namespace LoadRunnerClient.Messages
{
    public class SessionListMessage
    {
        public const string TYPE = "SessionListMessage";
        private List<SessionDTO> _listOfSessionDTOS;

        public SessionListMessage()
        {
        }

        public SessionListMessage(List<SessionDTO> sessions)
        {
            this._listOfSessionDTOS = sessions;
        }

        public List<SessionDTO> listOfSessionDTOS
        {
            get => _listOfSessionDTOS;
            set => _listOfSessionDTOS = value;
        }
    }
}